src/cloudservice/azext_cloudservice/manual/_help.py

```diff
@@ -84,7 +84,7 @@
 
 helps['cloud-service'] = """
     type: group
-    short-summary: Manage cloud service
+    short-summary: Manage cloud service (extended support). All commands in this extension are extended support.
```

Comment:
Should we just keep it as - "Manage cloud service (extended support)."?
The subsequent line doesn't seem to add much context or help clarify.

Reply:
Done